### PR TITLE
Handling missing translators

### PR DESCRIPTION
**Pull request by danielrischar**
Adds logging for the errors that might come from getPlatforms.

### CODE REVIEW

File: reference-app/app.js [hunk 1]
```diff
@@ -51,7 +51,9 @@ mainModule.controller('MainCtrl', ['$scope', '$http', '$q', 'remote', 'config',
             $scope.loading = true;
             getHubData(hub).then(platforms => {
                 $scope.selectedHub = hub;
-                $scope.selectPlatform(platforms[0]);
+                $scope.selectPlatform(platforms.platforms[0]);
+                // Log any failed translators
+                logInfo(JSON.stringify(platforms.errors, null, 2));
```
Comment (arjun-msft) [inline reference-app/app.js:56]: logWarn? #WontFix

File: reference-app/app.js [hunk 2]
```diff
@@ -65,13 +67,15 @@ mainModule.controller('MainCtrl', ['$scope', '$http', '$q', 'remote', 'config',
...
-                        $scope.selectPlatform(platforms[i]);
+                for (let i = 0; i < platforms.platforms.length; i++) {
+                    if (platforms.platforms[i].info.opent2t.controlId === currentPlatformId) {
+                        $scope.selectPlatform(platforms.platforms[i]);
                         break;
                     }
                 }
             }
+            // Log any failed translators
+            logInfo(JSON.stringify(platforms.errors, null, 2));
```
Comment (arjun-msft) [inline reference-app/app.js:78]: logWarn? #WontFix

**Comment by arjun-msft** [comment]
:shipit:

**Comment by shalinijoshi19** [comment]
:shipit:

**Review by shalinijoshi19 — APPROVED** [review]

File: reference-app/app.js (hunk 1 above)
Comment (danielrischar) [reply reference-app/app.js:56]: Reference app only currently supports info and error states.  We can add warning, but I don't think it's needed at the moment.

---
In reply to: [107044861](https://github.com/openT2T/opent2t-cli/pull/30#discussion_r107044861) [](ancestors = 107044861)

**Pull request merged**